iothub/service/src/DigitalTwin/readme.md

```diff
@@ -1,20 +1,91 @@
-#### Autorest config file for generating Digital Twin Service Client models and protocol layer
+  > The `DigitalTwinClient` is available only on .NET framework 4.7.2 and .NET Standard 2.0+.
 
-#### Protocol layer generation:
-- Run the powershell script `generateCode.ps`. It will pick up the config below and output the results into this folder. It will also make a few automated changes to the generated protocol layer, that are required for this client library.
+### Protocol layer generation:
+- Run the powershell script [generateCode.ps](./generateCode.ps1). It will pick up the autorest config and output the results into this folder. It will also make a few automated changes to the generated protocol layer, that are required for this client library.
```

Comment:
add link to readme.md ?

Reply:
I don't understand, which readme? This doc is the readme.

Reply:
Do you mean the autorest config?

Reply:
I assume you mean the autorest config yaml, so I'll add a link to that.